src/botPage/view/TradeInfoPanel/TradeTable.js

```diff
@@ -19,10 +20,18 @@ const getProfit = ({ sell_price: sellPrice, buy_price: buyPrice, currency }) =>
     return '';
 };
 
+const getTimestamp = date => {
+    const buyDate = new Date(date * 1000);
+    return `${buyDate.toISOString().split('T')[0]} ${buyDate.toTimeString().slice(0, 8)} ${
+        buyDate.toTimeString().split(' ')[1]
+    }`;
+};
+
 const minHeight = 290;
 const rowHeight = 25;
 
 const ProfitColor = ({ value }) => <div style={value > 0 ? style.green : style.red}>{value}</div>;
+const StatusColor = ({ value }) => <div style={value === 'Settled' ? style.green : style.red}>{value}</div>;
```

Comment:
Make sure to run the "_Settled_" string through the `translate` function (`src/common/i18n.js`).

Reply:
Aye, capt'n!